### PR TITLE
Allow renaming cluster, host, and storage

### PR DESCRIPTION
**Pull request by GabrielBrascher**
## Description

Current CloudStack master (4.15.0.0-SNAPSHOT) API commands `updateHost`, `updateCluster`, and `updateStoragePool` allow updating the respective resource name.

This PR adds support for such edit buttons on CloudStack primate.
CloudStack PR: https://github.com/apache/cloudstack/pull/4165

## Screenshots
Update Host name:
image

Update Cluster name:
image

Update Storage name:
image

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@GabrielBrascher Could you change `label.clustername` in en.json to `Cluster Name`

**Review by davidjumani — APPROVED** [review]
Breaking on primate QA, but tested locally. Working fine!

**Comment by davidjumani** [comment]
@rhtyd Good to go!

**Comment by GabrielBrascher** [comment]
@davidjumani thanks for the review. I pushed a commit changing `clustername.label` from `Cluster` to `Cluster Name`.

**Comment by GabrielBrascher** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
Thanks @GabrielBrascher 
@rhtyd Good to go!

File: src/config/section/infra/primaryStorages.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default {
       icon: 'edit',
       label: 'label.edit',
       dataView: true,
-      args: ['tags', 'capacitybytes', 'capacityiops']
+      args: ['name', 'tags', 'capacitybytes', 'capacityiops']
```
Comment (rhtyd) [inline src/config/section/infra/primaryStorages.js:64]: @davidjumani how does it fare if mgmt server/API does not support the arg?

Comment (davidjumani) [reply src/config/section/infra/primaryStorages.js:64]: it'll get passed but ignored by the backend, but yes, better add an ACS version check

**Pull request merged**